Gandalan.IDAS.WebApi.Client/BusinessRoutinen/WebRoutinenBase.cs

```diff
@@ -664,6 +694,123 @@ private void SetupRestRoutinen(RESTRoutinen cl)
             }
         }
 
+        private bool CheckJwtToken()
```

Comment:
The only difference between `CheckJwtToken` and `CheckJwtTokenAsync` is line with `var newJwt`.
Please reduce code duplication